src/Commands/ClientCreatorCommand.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Vinelab\ClientGenerator\Commands;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Vinelab\ClientGenerator\Storage\ClientStorage;
+
+class ClientCreatorCommand extends Command
+{
+    /**
+     * Command for creating a new client.
+     *
+     * @var string
+     */
+    protected $commandName = 'clients:create';
+
+    /**
+     * Client creation command description.
+     *
+     * @var string
+     */
+    protected $commandDescription = 'Generates a new client.';
+
+    /**
+     * The name argument.
+     *
+     * @var string
+     */
+    protected $commandAppName = 'name';
```

Comment:
why prefix the properties with `$commandApp`? it's client and not really necessary to prefix so we can use `$name` directly.